driver/lib/sessions/ios.ts

```diff
@@ -35,10 +32,18 @@ export const getObservatoryWsUri = async (proxydriver) => {
   if (realDevice) {
     // @todo check if `brew install usbmuxd` is needed
     log.info(`Running on iOS real device, doing "iproxy" now`);
-    const cmd = `iproxy ${urlObject.port} ${urlObject.port} ${udid}`;
-    log.debug(cmd);
-    await execPromise(cmd);
-    log.info(`"iproxy" successfully`);
+    const args = [urlObject.port, urlObject.port, udid];
+    log.debug(`iproxy ${urlObject.port} ${urlObject.port} ${udid}`);
+    const cmd = spawn(`iproxy`, args);
+    let stdout = ``;
+    cmd.stdout.on(`data`, (data) => {
+      stdout += data;
+      if (stdout.includes(`waiting for connection`)) {
+        log.info(`"iproxy" started successfully`);
+      } else {
+        log.debug(`"iproxy" not started successfully ${stdout}`);
+      }
+    });
```

Comment:
This block of code doesn't output anything because of this [issue](https://github.com/libimobiledevice/libusbmuxd/issues/79).  I made another [issue](https://github.com/libimobiledevice/libusbmuxd/issues/103) to follow up with.